MediaAdminBundle/Resources/public/less/mediaModalFolder.less

```diff
@@ -20,6 +20,12 @@
         }
 
         ul {
+            .active {
```

Comment:
It's to get the full tree for an easy maintenance: if we put ``.active > a`` now, there's a chance that another branch emerges beside later, for instance ``.active div``, instead of putting the div selector inside de .active slector.